Sources/FigSchema/FigScriptGenerator.swift

```diff
@@ -1,24 +1,31 @@
 import Foundation
 
-public struct FigScriptGenerator {
-    static let encoder: JSONEncoder = {
-        let encoder = JSONEncoder()
+public class FigScriptGenerator {
+    static let shared = FigScriptGenerator()
+
+    private var encoder: JSONEncoder!
+
+    init() {
+        setupEncoder()
```

Comment:
Consider setting up the encoder in `init` so that we don't have an unnecessary IUO